express/scripts/martech.js

```diff
@@ -516,40 +530,28 @@ loadScript('https://www.adobe.com/etc.clientlibs/globalnav/clientlibs/base/feds.
 loadScript('https://static.adobelogin.com/imslib/imslib.min.js');
 
 /* Core Web Vitals */
-const usp = new URLSearchParams(window.location.search);
-const cwv = usp.get('cwv');
-
-// with parameter, weight is 1. Defaults to 100.
-const weight = (cwv === 'on') ? 1 : 100;
-
-window.hlx = window.hlx || {};
-
-// eslint-disable-next-line no-bitwise
-const hashCode = (s) => s.split('').reduce((a, b) => (((a << 5) - a) + b.charCodeAt(0)) | 0, 0);
-const id = `${hashCode(window.location.href)}-${new Date().getTime()}-${Math.random().toString(16).substr(2, 14)}`;
-
-function store(data) {
-  const body = JSON.stringify(data);
-  const url = `https://rum.hlx3.page/.rum/${weight}`;
-
-  // console.log('storing', body);
-
-  // Use `navigator.sendBeacon()` if available, falling back to `fetch()`.
-  // eslint-disable-next-line no-unused-expressions
-  (navigator.sendBeacon && navigator.sendBeacon(url, body))
-  || fetch(url, { body, method: 'POST', keepalive: true });
-}
 
 function storeCWV(measurement) {
+  const { weight, id, generation } = window.hlx.rum;
   const rum = { cwv: { }, weight, id };
   rum.cwv[measurement.name] = measurement.value;
   rum.referer = window.location.href;
-  store(rum);
+  if (generation) rum.generation = generation;
+  sendRUMData(rum, weight);
 }
 
-if (Math.random() * weight < 1) {
+const { weight, id, random } = window.hlx.rum;
+const generation = 'instrumentation-test';
+window.hlx.rum.generation = generation;
```

Comment:
Same here.

Reply:
this one is used in the CWV submissions...